.all-contributorsrc

```diff
@@ -311,10 +311,20 @@
       "contributions": [
         "doc"
       ]
+    },
+    {
+      "login": "JReinhold",
+      "name": "Jeppe Reinhold",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/5678122?v=4",
+      "profile": "https://reinhold.is",
+      "contributions": [
+        "code"
+      ]
     }
   ],
   "files": [
     "README.md"
   ],
-  "repoHost": "https://github.com"
+  "repoHost": "https://github.com",
+  "commitConvention": "none"
```

Comment:
`commitConvention` needs to be set to `"angular"`.

Other than that it looks good to me.

Reply:
That's fine, it was just auto-generated by the CLI. I've also put the options above the contributors array, to make it easier to read.

Reply:
Okay, thank you.